Source/Core/VideoCommon/TextureCacheBase.cpp

```diff
@@ -1107,6 +1107,21 @@ void TextureCacheBase::CopyRenderTargetToTexture(u32 dstAddr, unsigned int dstFo
 		}
 	}
 
+	if (dstStride < bytes_per_row)
+	{
+		// This kind of efb copy results in a scrambled image.
```

Comment:
Has this been hardware verified? Have you actually looked at the code triggering these copies to figure out how it's computing the stride?